site/app/controllers/forum/ForumController.php

```diff
@@ -433,6 +445,7 @@ public function pinThread($type){
         $current_user = $this->core->getUser()->getId();
         $this->core->getQueries()->addPinnedThread($current_user, $thread_id, $type);
         $response = array('user' => $current_user, 'thread' => $thread_id, 'type' => $type);
+        $this->checkNotifyToUnPin($type);
```

Comment:
Do we want to notify the instructor every time they favorite(pin) a thread? The [issue](https://github.com/Submitty/Submitty/issues/4432) only mentions announcements.